Graph/lineMooring1DVar.m

```diff
@@ -60,13 +60,32 @@ function lineMooring1DVar(sample_data, varName, isQC, saveToFile, exportDir)
 stringQC = 'non QC';
 if isQC, stringQC = 'QC'; end
 
+% some variables (eg ECUR or CNDC) while qc for in/out water do not
+% have further processing and are marked raw. If varName is not
+% listed in imosGlobalRangeQC.txt chances are this is one such
+% variable.
+% read all values from imosGlobalRangeQC properties file
+values = readProperty('*', fullfile('AutomaticQC', 'imosGlobalRangeQC.txt'));
+specialVars = strtrim(values{1});
+% also test if variable_MAG (eg UCUR_MAG)
+tokens = regexp(varName,'([a-zA-Z0-9]+)(?:_MAG*)','tokens');
+if isempty(tokens), tokens{1} = ''; end
+if any(strcmpi(varName, specialVars)) | any(strcmpi(tokens{:}, specialVars))
+    useQCgood = true;  % test against qc=1 or 2
+else
+    useQCgood = false; % test against qc=0
+end
+
```

Comment:
Could you please explain me why you think it's useful to plot CNDC or ECUR across the mooring?

Not sure about ECUR...

Maybe CNDC, but then would it be useful to QC CNDC properly and starting by giving it a global range threshold test?